SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/storage-policies.md

--------------------

To retrieve the storage policies in an enterprise, call

> [`client.storagePolicies.list(marker: String?, limit: Int?, fields: [String]?)`][get-storage-policies].  This method will return an iterator in the completion that can be used to get the policies.

Reviewer: ```suggestion
[`client.storagePolicies.list(marker: String?, limit: Int?, fields: [String]?)`][get-storage-policies].  This method will return an iterator in the completion, which is used to get the policies.
```
